### PR TITLE
Update DDR !findmodules to output number of matches

### PR DESCRIPTION
**Pull request by acrowthe**
- Will allow for improved testing
- Clarifies when no results were found over previous empty output

Signed-off-by: Andrew Crowther <acrowthe3388@gmail.com>

### CODE REVIEW

**Comment by acrowthe** [comment]
@keithc-ca Could you please review this when you have time?

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/FindModulesCommand.java [hunk 1]
```diff
@@ -104,13 +104,16 @@ public void run(String command, String[] args, Context context, PrintStream out)
...
 			case MODULE:
-				ModularityHelper.iterateModules(out, moduleFilter, moduleOutput, filterArg);
+				result = ModularityHelper.iterateModules(out, moduleFilter, moduleOutput, filterArg);
+				out.printf("Found %d module%s\n", result, (1 == result ? "": "s"));
 				break;
 			case PACKAGE:
 				ModularityHelper.PackageOutput packageOutput = ModularityHelper::printPackageJ9Module; 
-				ModularityHelper.iteratePackages(out, packageFilter, packageOutput, filterArg);
+				result = ModularityHelper.iteratePackages(out, packageFilter, packageOutput, filterArg);
+				out.printf("Found %d module%s\n", result, (1 == result ? "": "s"));
```
[the remark this answers is not in the document]
Comment (acrowthe) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/FindModulesCommand.java]: This case actually outputs the module that owns a package, through `printPackageJ9Module`.
So, module is actually correct.

A comment has been added to explain this

[the remark this answers is not in the document]
Comment (acrowthe) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/FindModulesCommand.java]: The use case is:
I have a known package name that I want to find the module of.
```
> !findmodules package moduleA/packageA
moduleA                        !j9module <moduleAddress>
Found 1 module
```

**Comment by keithc-ca** [comment]
Jenkins test sanity zlinux jdk11

**Pull request merged**